### DISCUSSION TITLE
FAQ about Tasking Manager

### DISCUSSION THREAD

**Discussion by ramyaragupathy** [hotosm/tasking-manager#6228] [Ideas]
We're excited to announce that we are creating a comprehensive FAQ document to better serve our users and provide valuable information. We believe that your insights and questions will greatly enhance the content.

#### How can you contribute?

-  Access the collaborative FAQ document [on Google Docs here](https://docs.google.com/document/d/1ofcEbVGJOL7CMxcH4FV0toveoZTANGQEyvTbxcs8i2g/edit#heading=h.3wk5a3i5rc41).
-   Add your questions, suggestions, or edits directly as a comment to the document.
-  Feel free to comment on existing questions, providing additional context or feedback - I have added a few template questions to start with.

Together, we can build a resource that addresses the most common questions and concerns our mapping community may have. Your input is invaluable, and we appreciate your contribution to making Tasking Manager even more user-friendly.

**Comment by Adrian-Shobrooke** [comment]
Is this document linked from a sticky thread in Slack? Might get some more readers and perhaps some input.
